docs/spl-docs/evaluation-functions/comparison-conditional-functions.md

```diff
@@ -0,0 +1,177 @@
+# Comparison and Conditional functions
+
+_This list includes functions that can be used to return values based on conditions or compare different values._
+
+
+## case(\<condition\>, \<value\>, ...)
+This function takes in pairs of condition and value and returns the value corresponding to the first condition that evaluated to `true`.
+
+#### Usage
+- `<condition>` arguments are Boolean expressions that evaluates to `true` or `false`. 
+- `<value>` argument can be a string or a numeric literal. 
+- If all the conditions are `false`, the default value returned is `NULL`.
+- One can use this function with eval and where commands.
+
+#### Example
+The following command returns the performance by checking the sales_amount with different conditions and returns the performance value corresponding to the first condition evaluated to `true`. 
+For example, if sales_amount for a record is `11000`, the first condition will evaluate to true and `"Excellent"` would be returned. 
+If the sales_amount is `3000`, then first two conditions will evaluate to false and third condition will evaluate to `true` and `"Average"` would be returned.
+```
+... | eval sales_performance=case(sales_amount >= 10000, "Excellent", 
+                                    sales_amount >= 5000, "Good", 
+                                    sales_amount >= 1000, "Average", 
+                                    sales_amount > 0, "Below Average", 
+                                    sales_amount = 0, "No Sale")
+```
+
+
+## validate(\<condition\>, \<value\>, ...)
+This function is opposite of case function. It takes in pairs of condition and value and returns the value corresponding to the first condition that evaluated to `false`. 
+
+#### Usage
+- `<condition>` arguments are Boolean expressions that evaluates to `true` or `false`. 
+- `<value>` argument can either be a string or a numeric literal. 
+- If all conditions are `true`, the default value returned is `NULL`.
+- One can use this function with eval and where commands.
+
+#### Example
+The following command checks the field age and based on the condition returns appropriates values.
+For e.x. if value in the field age is not an integer, it would return `"Age is not an integer"`.
+```
+... | eval is_valid=validate(isint(age), "Age is not an integer", 
+                               age < 18 OR age > 120, "Age is not valid")
+```
```

Comment:
nit: this isn't rendering nicely. You might want to have it all on one line, or pad the second line differently
<img width="991" alt="image" src="https://github.com/siglens/siglens-docs/assets/20426590/eab2fd9f-ac34-4ba8-aeda-ee0608fb0fe2">

Reply:
nit: I don't think this example quite makes sense, because if the age is valid, the entry for the `is_valid` field for that row will be `NULL`